srcpkgs/avr-gdb/template

```diff
@@ -1,79 +1,38 @@
 # Template file for 'avr-gdb'
 pkgname=avr-gdb
 version=10.1
-revision=1
+revision=2
 wrksrc=gdb-${version}
 build_style=gnu-configure
-pycompile_dirs="/usr/share/gdb/python"
-configure_args="\
-	--target=avr \
-	--program-prefix=avr- \
-	--disable-werror \
-	--disable-nls \
-	--with-system-readline \
-	--with-system-gdbinit=/etc/gdb/gdbinit \
-	--with-system-zlib $(vopt_enable gdbserver) \
-	$(vopt_if static 'CFLAGS=-static CXXFLAGS=-static LDFLAGS=-static') \
-	$(vopt_if python --with-python=/usr/bin/python3)"
-hostmakedepends="texinfo perl $(vopt_if python python3-devel)"
-makedepends="ncurses-devel zlib-devel readline-devel expat-devel
+configure_args="--disable-werror --disable-nls --with-system-readline
+ --with-system-gdbinit=/etc/gdb/gdbinit --with-system-zlib
+ $(vopt_with guile) $(vopt_if python --with-python=/usr/bin/python3)
+ --target=avr"
```

Comment:
For consistency, I'd rather have target as the first configure arg and leave the optional ones for last; this applies for all the templates.